hangman.js

```diff
@@ -0,0 +1,147 @@
+  let css = document.querySelector("link")
+
+  //Setting variables for tags on my HTML page
+  let header1 = document.querySelector('h1')
+  let form = document.querySelector('form')
+  let first_button = document.getElementById('Guess-Button')
+  let body = document.querySelector('body')
+  let p = document.querySelector('p')
+  let rule1 = document.getElementById('item1')
+  let rule2 = document.getElementById('item2')
+  let rule3 = document.getElementById('item3')
+  let rule4 = document.getElementById('item4')
+  let ul = document.querySelector('ul')
+  let rule_paragraph = document.getElementById('rules')
+
+  //My array of mystery words
+  let arr = ['Avengers', 'Titanic', 'Aladdin', 'Godzilla', 'Jaws']
+  const wordGrabber = (arr) => arr[Math.floor((Math.random() * arr.length))]
+  let mysteryWord = wordGrabber(arr)
+  let rightGuesses = [];
+  let wrongGuesses = [];
+  let allGuesses = [];
+  let guessesLeft = 6
+
+// A function for when the player wins the game
+  function win() {
+    window.alert(`You Have Won! Your Word Was ${mysteryWord}. Play Again!`)
+    header1.innerText = `You've Saved Us All.`
+    form.removeChild(guess)
+    form.removeChild(first_button)
+    para3.innerText = (`Your word was ${mysteryWord}. GOOD JOB!`)
+    document.body.appendChild(para3)
+  }
+
+// A function for when the player loses
+  function lose() {
+    header1.innerText = 'The Asteroid Has Arrived'
+    css.setAttribute("href", "lose.css")
+    window.alert(`You Have Lost. Your Word Was ${mysteryWord}. Play Again?`)
+    form.removeChild(guess)
+    form.removeChild(first_button)
+    ul.removeChild(rule1)
+    ul.removeChild(rule2)
+    ul.removeChild(rule3)
+    ul.removeChild(rule4)
+    form.removeChild(rule_paragraph)
+    para3.innerText =
+    (`You ran out of time.
+    Your word was: ${mysteryWord.toUpperCase()}.
+    Luckily the asteroid just barely missed.`)
+    document.body.appendChild(para3)
+  }
+
+  for (let i = 0; i < mysteryWord.length; i++) {
+      rightGuesses[i] = "_";
+  }
+
+  let remainingLetters = mysteryWord.length
+
+
+  let para1 = document.createElement('p')
+  para1.innerText = rightGuesses.join(" ")
+
+  let para2 = document.createElement('p')
+  para2.innerText = (`You Have ${guessesLeft} Guesses Left`)
+
+  let para3 = document.createElement('p')
+  let para4 = document.createElement('p')
+  para4.innerText = `Wrong Answers: ${wrongGuesses}`
+
+
+  document.body.appendChild(para1)
+  document.body.appendChild(para2)
+  document.body.appendChild(para4)
+
+
+  document.addEventListener('submit',()=>{
+    event.preventDefault()
+    guess = document.querySelector('#Guess')
+    let string_checker = /^[a-zA-Z]+$/
+    p.innerText = ''
+
+    while (!allGuesses.includes(guess.value.toUpperCase())) {
+
+      for (let i = 0; i < mysteryWord.length; i++) {
+        if(guess.value.trim() === ''){
+          window.alert(`You Entered Nothing. Please Enter a Valid Guess. Letters Only!`)
```

Comment:
One glitch here is that when the user leaves the input form blank, window.alert pops up indefinitely which traps the user in loop.